src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.constClimber;
+import frc.robot.RobotMap.mapClimber;
+
+public class Climber extends SubsystemBase {
+  TalonFX climberMotor;
+  TalonFXConfiguration climberConfig;
+
+  /** Creates a new Climber. */
+  public Climber() {
+    climberMotor = new TalonFX(mapClimber.CLIMBER_MOTOR_CAN, "rio");
+
```

Comment:
Remember to reset to factory defaults by creating and applying a TalonFXConfiguration